ios/RCTMapboxGL/RCTMapboxGL.m

```diff
@@ -381,6 +383,12 @@ - (void)removeAnnotation:(NSString*)selectedIdentifier
     }
 }
 
+- (void) setContentInset:(UIEdgeInsets)inset
+{
+    _inset = inset;
```

Comment:
This class already has a `contentInset` property and thus a `_contentInset` ivar. You should update `_contentInset` and get rid of the `_inset` ivar (also above when you construct the map).